test/test_art.py

```diff
@@ -210,19 +208,6 @@ def test_main_interface_falls_back_to_aao(self):
         self.plugin.art_for_album(album, [self.dpath])
         self.assertEqual(responses.calls[-1].request.url, self.AAO_URL)
 
-    def test_main_interface_uses_caa_when_mbid_available(self):
-        self.mock_response("http://" + self.CAA_URL)
-        self.mock_response("https://" + self.CAA_URL)
-        album = _common.Bag(mb_albumid=self.MBID, asin=self.ASIN)
-        candidate = self.plugin.art_for_album(album, None)
-        self.assertIsNotNone(candidate)
-        self.assertEqual(len(responses.calls), 1)
-        if util.SNI_SUPPORTED:
-            url = "https://" + self.CAA_URL
-        else:
-            url = "http://" + self.CAA_URL
-        self.assertEqual(responses.calls[0].request.url, url)
-
```

Comment:
I think this test should stay: The intent is slightly different from the one added in `CoverArtArchiveTest` since here, precedence of the art sources is tested.

Reply:
It probably should stay but I couldn't find a simple way to mock the tests that didn't involve just copy/pasting the two large JSON responses that need to be mocked.

I'm open to ideas.

Reply:
I can't think of anything pretty offhand, either. But maybe just referencing the class members of `CoverArtArchiveTest` would be fine? Or a mixin class (`CoverArtArchiveTestData` for lack of a better name) could be added that contains just the URLs,  MBIDs and RESPONSEs.

Reply:
The latest commit adds a mixin class and brings back this test.